lib/inspec/control_eval_context.rb

```diff
@@ -35,7 +35,7 @@ def self.rule_context(resources_dsl, profile_id)
             options[:priority] = 20
             options[:provider] = :inline_control_code
             evt = Inspec::Input.infer_event(options)
-            Inspec::InputRegistry.find_or_register_input(input_name, profile_name, event: evt).value
+            Inspec::InputRegistry.find_or_register_input(input_name, profile_id, event: evt).value
```

Comment:
This doesn't track, so I assume it is the subject of the question in the commit message.

Is it possible / worthwhile to write a test to reproduce this failure (and thus prevent regressions)?

Reply:
This codepath is well exercised by tests - anytime we make an input / attribute with options (for example, a value) this should get touched. For example, the 'scoping' test profile tests this.

I really don't know how this was working at all when we merged in the inputs overhaul. Not reassuring, but it is well-tested at this point.